modules/core/src/array.cpp

```diff
@@ -941,7 +944,10 @@ cvSetData( CvArr* arr, void* data, int step )
             img->widthStep = min_step;
         }
 
-        img->imageSize = img->widthStep * img->height;
+        const int64 imageSize_tmp = (int64)img->widthStep*(int64)img->height;
+        img->imageSize = imageSize_tmp;
```

Comment:
Could you please add explicit type cast here to fix warnings produced by MSVS compiler?